iree/compiler/Codegen/Common/test/iree_comprehensive_bufferize.mlir

```diff
@@ -40,7 +40,8 @@ func @matmul() {
 //  CHECK-DAG:   %[[M:.+]] = hal.interface.constant.load[0]
 //  CHECK-DAG:   %[[N:.+]] = hal.interface.constant.load[1]
 //  CHECK-DAG:   %[[K:.+]] = hal.interface.constant.load[2]
-//  CHECK-DAG:   %[[LHS:.+]] = hal.interface.binding.subspan set(0) binding(0) type(storage_buffer)
+//  CHECK-DAG:   %[[LHS:.+]] = hal.interface.binding.subspan set(0) binding(0) type(storage_buffer) alignment(32)
+//      CHECK:   memref.assume_alignment %[[LHS]], 32
```

Comment:
Maybe CHECK-DAG could fix the issue

Reply:
Turns out I put changed two different test case ;)

Reply:
Turns out I put changed two different test case ;)